core/src/sumeragi/network_topology.rs

```diff
@@ -228,13 +228,16 @@ impl Topology {
     /// Apply new committed block hash.
     #[allow(clippy::expect_used)]
     pub fn apply_block(&mut self, block: HashOf<VersionedCommittedBlock>) {
+        /*
+        nocheckin
```

Comment:
So is this not supposed to be checked into `stating` or `dev`?

Reply:
Not to dev. Originally it was a note to me to remember to reimplement view changes. But instead I decided that this can live on the staging branch and there is a issue tracking it. https://github.com/hyperledger/iroha/issues/2561

Reply:
yes